src/clj/pyregence/capabilities.clj

```diff
@@ -286,8 +286,8 @@
   (let [match-drop-names (->> (call-sql "get_user_match_names" user-id)
                               (reduce (fn [acc row]
                                         (assoc acc (:match_job_id row)
-                                                   (str (:display_name row)
-                                                        " (Match Drop)")))
+                                               (str (:display_name row)
+                                                    " (Match Drop)")))
```

Comment:
Same comment here as above: should we move `(:match_job_id row)` down to its own line?